apps/web/src/providers/feature-flag.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+
+import { createContext, useCallback, useContext, useEffect, useState } from 'react';
+
+import {
+  FEATURE_FLAG_POLL_INTERVAL,
+  LOCAL_FEATURE_FLAGS,
+  isFeatureFlagEnabled,
+} from '@documenso/lib/constants/feature-flags';
+
+import { getAllFlags } from '~/helpers/get-feature-flag';
+
+export type FeatureFlagValue = boolean | string | number | undefined;
```

Comment:
I noticed we have Zod for this elsewhere without being explicitly named, seems like we can tidy this into a `ZFeatureFlagValue` and `TFeatureFlagValue` combo for usage in the api route.

Reply:
Updated to use Zod, but I couldn't find other instances where it was being duplicated.